test/test_model.py

```diff
@@ -596,6 +596,60 @@ def test_remote_app_status(self):
             ['relation-list', '-r', '4', '--format=json'],
         ])
 
+    def test_storage(self):
+        meta = op.charm.CharmMeta()
+        meta.storages = {'disks': None, 'data': None}
+        self.model = op.model.Model('myapp/0', meta, self.backend)
+
+        self.assertEqual(len(self.model.storages), 2)
+        self.assertEqual(self.model.storages.keys(), meta.storages.keys())
+        self.assertTrue('disks' in self.model.storages)
+
+        fake_script(self, 'storage-list', """[ "$1" = disks ] && echo '["disks/0", "disks/1"]' || echo '[]'""")
+        fake_script(self, 'storage-get',
+                    """
+                    if [ "$2" = disks/0 ]; then
+                      echo '"/var/srv/disks/0"'
+                    elif [ "$2" == disks/1 ]; then
```

Comment:
== or =?  I believe = is the most natural here, as it's not a numeric comparison.